db/migrate/20190523115648_rename_timeshare_value_column_for_other_assets_declarations.rb

```diff
@@ -0,0 +1,5 @@
+class RenameTimeshareValueColumnForOtherAssetsDeclarations < ActiveRecord::Migration[5.2]
+  def change
+    rename_column :other_assets_declarations, :timeshare_value, :timeshare_property_value
```

Comment:
You can group this migration and the other one about `jewellery_value` in the same file. No need to have 2.

Reply:
You can delete `20190516111339_rename_jewellery_value_column_for_other_assets_declarations.rb`, rename this one to `20190523115648_rename_other_assets_declarations_columns` for example.

and put the two lines together:
```ruby
class RenameOtherAssetsDeclarationsColumns < ActiveRecord::Migration[5.2]
  def change
    rename_column :other_assets_declarations, :jewellery_value, :valuable_items_value
    rename_column :other_assets_declarations, :timeshare_value, :timeshare_property_value
  end
end
```